tests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/application/v3/advanced/LogsTest.java

```diff
@@ -66,8 +61,11 @@ public void testPodLogOfApplicationPod() {
 		
 		ConsoleView consoleView = new ConsoleView();
 		
-		new WaitUntil(new ConsoleHasText(), TimePeriod.NORMAL, false);
-		new WaitUntil(new ConsoleHasNoChange(TimePeriod.getCustom(5)), TimePeriod.VERY_LONG);
+		new WaitUntil(new ConsoleHasText() {
+			@Override public String errorMessage() {
+				return "Console of pod log is not opened";
+			}}, TimePeriod.NORMAL, false);
```

Comment:
Is the errorMessage even used, when the third parameter is "false"? It means the exception won't be thrown and I think that the exception message is the only place where errorMessage is used.